### DISCUSSION TITLE
Why is v1 the default branch?

### DISCUSSION THREAD

**Discussion by quinnypig** [aws/aws-cli#7573] [Q&A] [answered]
Increasingly v2 is what's being referenced in formal AWS documentation, community posts, etc. To quote the official documentation, "some features introduced in version 2 are not backported to version 1 and you must upgrade to access those features."

Bearing that in mind, why is v1 the default branch on GitHub? I've tripped over this a few times, and I (mostly) know what's going on here. I can't shake the feeling that this is going to catch out newcomers at a fairly steady clip. 

Is there a plan or timeline for making v2 the default branch?

**Comment by tim-finnigan** [comment, marked as answer]
Hi @QuinnyPig thanks for reaching out. The AWS CLI team is in the process of developing a deprecation plan for v1. There will be an official announcement once that plan is finalized. Making v2 the default branch on GitHub will be a priority as part of that plan. There are some steps involving the release process that will need to be updated prior to then.

**Reply by quinnypig** [reply]
Cool! So… it's been four years. Why is v1 the default branch?
